### ISSUE TITLE
Downloading worksheets can fail with non-ASCII characters

### ISSUE THREAD

**Issue by kcrisman** [sagemath/sagenb#410]
Apparently [Trac 9711](https://trac.sagemath.org/ticket/9711) was never quite fixed - see [here](https://groups.google.com/forum/#!topic/sage-support/Ej3yNTYLh9A) and [this ask.sagemath question](http://ask.sagemath.org/question/34631/500-internal-server-error).

**Comment by fchapoton** [comment]
This seems to work with the latest update of sagenb, see https://trac.sagemath.org/ticket/9711

**Comment by embray** [comment]
Indeed, I tested it in Sage 8.3.  I tried putting non-ASCII text all over the place, and have tried different methods of downloading and uploading worksheets, etc. and can no longer reproduce the problem.  I think it was probably fixed, at least for the most part, by 912640f08d8b6253967335675f11b4f83088cb43

Unless it comes up again and someone can provide specific steps for reproducing the issue we should call this fixed.
